be/src/column/nullable_column.cpp

```diff
@@ -347,7 +347,7 @@ void NullableColumn::fnv_hash(uint32_t* hash, uint32_t from, uint32_t to) const
         return;
     }
 
-    auto null_data = _null_column->get_data();
+    const auto& null_data = _null_column->get_data();
```

Comment:
I found that crc32_hash is also copying null_data, could you change it together?

Reply:
Good point. Done.